src/main/java/codelook/jpa/objects/Controller.java

```diff
@@ -6,11 +6,16 @@
 
 @org.springframework.stereotype.Controller
 public class Controller {
```

Comment:
clearer name + seperation would be nice - this looks like a view controller (see #21 )

also possibly consider splitting into a RESTController and ViewController so the REST methods can be reused for SPA methods via ajax

Reply:
agreed this can be resolved by those who will work on the pages, this was just an initial creation choice